microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+
+## Potential Misuse 
+We believe that “limited” is the right default that allows the most customer’s regional preferences to be respected with the least amount of risk of the website misusing this information. To understand why these options exist, one must understand a bit about the ways in which regional information expressed by the OS can be misused by websites. 
+
+One way in which the OS regional information could be misused is to compromise the user’s privacy—the regional information would be one additional bit of fingerprinting entropy. In combination with many other preexisting bits of entropy in the platform today, the user could be uniquely identified leading to more potential secondary harms. It’s important to remember that a user’s region and language are already available via the web platform, but users generally fall into well-recognized regional codes that are similar across geographies, and because the sample sizes are so large, the risk of any specific regional code “standing out” is small and thus the risk is small. If the user changes their OS preferences to a unique language + region, websites may be able to identify the anomaly in a population, and easily uniquely identify the associated customer. 
+
+Another way in which the OS regional information could be misused by a website is in the consistency of their content presentation. There are various places in the web platform where language and regional information is made available. Regional information is processed in one place, while language preferences are surfaced in another—furthermore, in a third location, this information was proactively sent to the server on the initial request for content. When the language and regional information is self-consistent, then it doesn’t really matter which signals the website uses to assemble its content presentation. However, if one part of the web platform starts reporting one language preference, but another part of the platform reports a separate language and regional preference, the website may start to mix-and-match regional preferences or language preferences, resulting in strange and unusual [broken] experiences, e.g., some times and dates formatted in one way, while other times and dates (from the same site!) formatted differently, or phrasing of requests that mixes languages together: “Email sent on miércoles, 5 de enero de 2021”.
+
+## Three settings
+Customers have three settings they can chose from:
+
+* Never 
+* Limited (default) 
+* Always (not recommended). 
+
+To minimize the risks noted previously, customers can disable the sharing of the OS regional information by choosing “never”. This does not prevent websites from reading region information, but does block any specific OS regional customizations that the user has made from being exposed to website content. Instead, the browser’s display language is used to derive a default region. 
```

Comment:
i might try 3 h4 headings, if the 3 settings paragraphs are independently broken out

Reply:
I did this rather than the above.